src/test/testRunner.ts

```diff
@@ -0,0 +1,67 @@
+import * as path from 'path';
+import * as cp from 'child_process';
+import {
+  runTests,
+  downloadAndUnzipVSCode,
+  resolveCliPathFromVSCodeExecutablePath,
+} from 'vscode-test';
+
+async function main(): Promise<void> {
+  try {
+    const executable = await downloadAndUnzipVSCode();
+    const cliPath = resolveCliPathFromVSCodeExecutablePath(executable);
+
+    // Install the latest released redhat.ansible extension
+    const installLog = cp.execSync(
+      `"${cliPath}" --install-extension redhat.ansible --force`
```

Comment:
Why is this necessary?

Reply:
When test starts for the first time and vscode is downloaded in the space, installations of the extensions need to be performed so that they can be tested. Plus `--force` flag is necessary so that the latest version of the extension is downloaded and all testings can be performed on it.